install.d/50-dracut.install

```diff
@@ -3,8 +3,8 @@
 COMMAND="${1:?}"
 KERNEL_VERSION="${2:?}"
 #shellcheck disable=SC2034
-BOOT_DIR_ABS="$3"
-KERNEL_IMAGE="${4:?}"
+BOOT_DIR_ABS="${3:?}"
```

Comment:
We don't actually use `BOOT_DIR_ABS` in this script, since the images we generate are copied to the staging area. So we don't have to enforce that $3 is set.

This is also why we have `#shellcheck disable=SC2034` here, shellcheck has a minor complaint about the variable not being used otherwise. For symmetry reasons I kept it in anyway, we do use it in `51-dracut-rescue.install`.

Reply:
That makes sense. Fixed!